### PR TITLE
Homepage -> Header 

### PR DESCRIPTION
**Pull request by emrerdem1**
This is Burak's branch that includes Header with background video.

### CODE REVIEW

**Comment by louisrli** [comment]
please have him pull request it himself

1 Ağu 2020 Cmt, saat 15:26 tarihinde emrerdem1 <notifications@github.com>
şunu yazdı:

File: src/components/header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of type
+            and scrambled it to make a type specimen book. It has survived not
+            only five centuries.
+          </p>
+          <div className="middleButtons">
+            <Button
+              variant="info"
+              style={{
```
Comment (louisrli) [inline src/components/header/Header.jsx]: don't use inline styles

File: src/components/header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            >
+              Get Started
+            </Button>
+            <Button
+              variant="secondary"
+              className="ml-5"
+              style={{
+                backgroundColor: "#C3C9EB",
+                borderColor: "#C3C9EB",
+                color: "#485181",
```
Comment (louisrli) [inline src/components/header/Header.jsx]: especially see how the color is duplicated 3 times when you can use a sass variable (and if its common throughout the app you should be using the same one in a common file)

File: src/components/header/Header.scss [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+.lineWrapper {
+  grid-column: 1/2;
+  grid-row: 5/6;
+  @media (max-width: 968px) {
+    display: none;
+  }
+}
+
+.content {
```
Comment (louisrli) [inline src/components/header/Header.scss]: need a better classname than `.content` or `.line` because it should easily clash

**Review by louisrli — APPROVED** [review]

**Review by OmarAlithawi — APPROVED** [review]

File: src/components/header/Header.scss [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+      padding-left: 2rem;
+      padding-right: 2rem;
+    }
+  }
+
+  .bottomButtons {
+    grid-column: 1/13;
+    grid-row: 12;
+  }
+}
```
Comment (emrerdem1) [inline src/components/header/Header.scss:156]: Line-break UwU

**Pull request merged**